openpype/hosts/blender/plugins/publish/validate_setdress_cam_instance.py

```diff
@@ -0,0 +1,49 @@
+from typing import List
+
+import bpy
+import string
+
+import pyblish.api
+
+import openpype.hosts.blender.api.action
+from openpype.pipeline.publish import ValidateContentsOrder
+
+
+class ValidateSetdressCamInstance(pyblish.api.Validator):
+    """Validate that cam and review instances points on the right collection."""
+
+    order = ValidateContentsOrder
+    hosts = ["blender"]
+    families = ["setdress"]
```

Comment:
you can deal with `camera` and `review` here instead of iterating `bpy.context.scene.openpype_instances`

Reply:
What do you mean when you say with `camera` and `review` ?